src/intl/IntlLocale.cpp

```diff
@@ -368,5 +395,268 @@ IntlLocaleObject::IntlLocaleObject(ExecutionState& state, Object* proto, String*
     m_locale = localeBuilder.finalize();
 }
 
+Value IntlLocaleObject::calendars(ExecutionState& state)
+{
+    ValueVector resultVector;
+    if (m_calendar) {
+        resultVector.pushBack(m_calendar.value());
+    } else {
```

Comment:
Suggest to reduce unnecessary indentation.(it is not required)

```suggestion
        resultVector.pushBack(m_calendar.value());
        return Object::createArrayFromList(state, resultVector);
    }
```

Reply:
both if-else needs  return Object::createArrayFromList(state, resultVector); statement

Reply:
yes, so I added return statement in suggested change

Reply:
contents of if also needs same statement